src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,38 @@
+package duke.command;
+
+import java.io.IOException;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Task;
+
+/**
+ * Represents the command to add Todo, Deadline and Event tasks.
+ */
+public class AddCommand extends Command {
+    private Task t;
+
```

Comment:
Perhaps you can name "Task t" to something like "Task task" as variables with large scope should have long names